### DISCUSSION TITLE
Transactions being imported with today's date

### DISCUSSION THREAD

**Discussion by harpoo123** [firefly-iii/firefly-iii#6522] [Firefly III]
Hi,

I am having the same issue posted below and I was wondering if you managed to solve it.

I map the transaction date field to Date but it still import everything with the same date.

Thanks

==============================================================
        I'm having this same issue running 4.7.17.3 in a docker.  I did it several times just to make sure I wasn't overlooking something.  I tried selecting + mapping the Date field and some of the other Transaction Date fields but they all come through with todays date.

![image](https://user-images.githubusercontent.com/26230753/62016549-68907400-b178-11e9-89cb-5de706ea85d1.png)

_Originally posted by @oldgeezy in https://github.com/firefly-iii/firefly-iii/issues/2178#issuecomment-515816246_

**Comment by JC5** [comment]
[This is probably the solution, let me know.](https://docs.firefly-iii.org/data-importer/usage/configure/#date-format)
